synapse/lib/stormlib/model.py

```diff
@@ -255,6 +255,14 @@ class LibModel(s_stormtypes.Lib):
                   'returns': {'type': ['model:tagprop', 'null'],
                               'desc': 'The ``model:tagprop`` instance of the tag prop if present or null.',
                               }}},
+        {'name': 'edge', 'desc': 'Get an edge object by name.',
+         'type': {'type': 'function', '_funcname': '_methEdge',
+                  'args': (
+                      {'name': 'edgetype', 'type': 'list', 'desc': 'The edge (n1form, verb, n2form) to retrieve.', },
```

Comment:
Should this be broken out into separate arguments?

Reply:
I wasn't sure but yeah probably should be, I'll adjust it